django/utils/autoreload.py

```diff
@@ -280,8 +280,14 @@ def reloader_thread():
 
 
 def restart_with_reloader():
+    import django.__main__
     while True:
-        args = [sys.executable] + ['-W%s' % o for o in sys.warnoptions] + sys.argv
+        args = [sys.executable] + ['-W%s' % o for o in sys.warnoptions]
+        if sys.argv[0] == django.__main__.__file__:
+            args += ["-m", "django"]
```

Comment:
Use single quotes (unless a string contains a single quote) as described in [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).

Perhaps a comment would help readability here?